packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -198,12 +200,20 @@ module('Integration | operator-mode', function (hooks) {
         </template>
       }
     );
+
     await waitFor('[data-test-person]');
     assert.dom('[data-type-display-name]').hasText('Person');
     assert.dom('[data-test-person]').hasText('Fadhlan');
     assert.dom('[data-test-first-letter-of-the-name]').hasText('F');
     assert.dom('[data-test-city]').hasText('Bandung');
     assert.dom('[data-test-country]').hasText('Indonesia');
+    assert.dom('.operator-mode-card-stack__card').exists({ count: 1 });
+
+    await click('[data-test-cardstack-operator-mode-overlay-button]');
+    assert.dom('.operator-mode-card-stack__card').exists({ count: 2 });
+    assert
```

Comment:
After adding these test assertions to assert a new card opens on top of the stack, an error pops up: `throw new Error(`bug: should never get here`);`

It's coming from here: https://github.com/cardstack/boxel/blob/dfc8df68cd6deed7a9e9111ab4c64f0eff6025c9/packages/host/app/components/operator-mode.gts#L72

It's because `relativeTo` is undefined. What should the card be relative to? Maybe I'm missing something in the test setup?

Reply:
I think this is a symptom of how these tests setup the cards. normally cards are deserialized as we recieve them from the server. the process of deserializing a card permits us to say what URL the card is relative to (which is generally the `id` field of the card). However, this way of instantiating a card skips a lot of those normal steps in the interest of test efficiency. specifically in the `loadCard()` helper in this module, the `createFromSerialized()` is called _without_ a relativeTo param. https://github.com/cardstack/boxel/blob/main/packages/host/tests/integration/components/operator-mode-test.gts#L47. we could fix this by just setting the card's ID as the relativeTo value

Reply:
Thanks! It turned out I also needed to pass `relativeTo` in `loadMissingField` - https://github.com/cardstack/boxel/pull/326/commits/9ca7fbdc8befb8f678e441a51ff0502851394b74